wapm.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "wabt"
+version = "1.0.12"
+description = "The WebAssembly Binary Toolkit"
+readme = "README-wapm.md"
+repository = "https://github.com/wapm-packages/wabt"
+
+[[module]]
+name = "wat2wasm"
+source = "out/wasi/Debug/wat2wasm"
+abi = "wasi"
+interfaces = {wasi= "0.1.0-unstable"}
+
+[[module]]
+name = "wast2json"
+source = "out/wasi/Debug/wast2json"
+abi = "wasi"
+interfaces = {wasi= "0.1.0-unstable"}
+
+[[module]]
+name = "wasm2wat"
+source = "out/wasi/Debug/wasm2wat"
+abi = "wasi"
+interfaces = {wasi= "0.1.0-unstable"}
+
+[[module]]
+name = "wasm-interp"
+source = "out/wasi/Debug/wasm-interp"
+abi = "wasi"
+interfaces = {wasi= "0.1.0-unstable"}
+
+[[module]]
+name = "wasm-validate"
+source = "out/wasi/Debug/wasm-validate"
+abi = "wasi"
+interfaces = {wasi= "0.1.0-unstable"}
+
+[[module]]
+name = "wasm-strip"
+source = "out/wasi/Debug/wasm-strip"
+abi = "wasi"
+interfaces = {wasi= "0.1.0-unstable"}
+
+# The commands for each module
+[[command]]
+name = "wat2wasm"
+module = "wat2wasm"
+
+[[command]]
+name = "wast2json"
+module = "wast2json"
+
+[[command]]
+name = "wasm2wat"
+module = "wasm2wat"
+
+[[command]]
+name = "wasm-interp"
+module = "wasm-interp"
+
+[[command]]
+name = "wasm-validate"
+module = "wasm-validate"
+
+[[command]]
+name = "wasm-strip"
+module = "wasm-strip"
```

Comment:
We probably want to add the wit-bindings as well?

Reply:
I thought commands don't have bindings because they're intended as executables that people run, not libraries that people import?

Reply:
Packages can have both commands and libraries.

That means, that a package can have 3 CLI binaries (with WASI), and 1 library (with WIT), for example.

So basically, we can have a new `libwabt` module that has bindings in the package :)

Reply:
Oh, that was [already added](https://github.com/Michael-F-Bryan/wabt/blob/ea60d90ca11f3763bb3737a7e7b6a06580f61c11/wapm.toml#L8-L13) yesterday before marking the PR ready. I'm guessing I forgot to push the code up.